src/sentry/sentry_apps/tasks/sentry_apps.py

```diff
@@ -388,17 +388,26 @@ def clear_region_cache(sentry_app_id: int, region_name: str) -> None:
 )
 @retry_decorator
 def workflow_notification(
-    installation_id: int, issue_id: int, type: str, user_id: int, *args: Any, **kwargs: Any
+    installation_id: int, issue_id: int, type: str, user_id: int | None, *args: Any, **kwargs: Any
 ) -> None:
-    webhook_data = get_webhook_data(installation_id, issue_id, user_id)
-    if not webhook_data:
-        return
-    install, issue, user = webhook_data
-    data = kwargs.get("data", {})
-    data.update({"issue": serialize(issue)})
-    send_webhooks(installation=install, event=f"issue.{type}", data=data, actor=user)
+    try:
+        event = SentryAppEventType(f"issue.{type}")
+    except ValueError as e:
+        raise SentryAppSentryError(message=SentryAppWebhookFailureReason.INVALID_EVENT) from e
```

Comment:
are we okay with this being outside of the context manager? i'm not sure if there's another way

Reply:
despairge we could add the smallest slo ever just for validate event but I think sentry error is sufficient

Reply:
yeah should be fine